wrappers/CMakeLists.txt

```diff
@@ -46,6 +46,9 @@ if(APPLE)
   endif()
 elseif(CMAKE_SYSTEM_NAME MATCHES "^Windows")
   add_definitions(/DWIN32_LEAN_AND_MEAN)
+  add_compile_options(
+      /MP # Enable multi-processor compilation
```

Comment:
@fealebenpae added this since it seemed like we didn't compile the wrapper sources in parallel.